### PR TITLE
Filter Extensions (DDL Providers)

### PR DESCRIPTION
**Pull request by Teekzilla**
Added a utility function to filter DDL links as sometimes other extensions are picked up. Currently, this is only checking for MKV, MP4, and AVI. This can be discussed to further expand however I don't know if it's necessary currently as all the content we retrieve have those extensions.

There was a point made that we could include all video file extensions because otherwise, down the line we wouldn't know if a provider starts using a different video format. Thus, I'll mark this for discussion as well as review and testing.

### CODE REVIEW

**Comment by SpajicM** [comment]
Not exactly how I imagined it. 

Instead of modifying every provider, I thought it should be called around here: https://github.com/ApolloTVofficial/Claws/blob/master/src/utils/WsWrapper.js#L45

**Comment by Teekzilla** [comment]
But my point on that is that this isn't necessary for normal providers. If we put it there it will apply to everything. I could add a check for `isDDL` there I suppose?

**Comment by SpajicM** [comment]
That would be better imo

**Comment by Teekzilla** [comment]
Fair enough. You're right, that would avoid another step in new providers. I'll update in a few.

**Comment by Teekzilla** [comment]
Updated based on the dialogue above.

**Comment by SpajicM** [comment]
Looks okay now

File: src/utils/createEvent.js [hunk 1]
```diff
@@ -24,7 +24,8 @@ function createEvent(data, ipLocked, pairing, { quality, provider, source, isRes
             quality,
             provider,
             source,
-            cookie
+            cookie,
+            isDDL
```
Comment (ddivad) [inline src/utils/createEvent.js]: "Technically" we don't care about isDDL here, right? As DDL's can only come from `result` events?

**Comment by ddivad** [comment]
Github has no "Approved with suggestions / questions" button, but LGTM. Just added a comment out of curiosity really.

**Pull request merged**